kvdb-rocksdb/src/lib.rs

```diff
@@ -448,12 +532,21 @@ impl Database {
 			Some(ref cfs) => {
 				let mut batch = WriteBatch::default();
 				let ops = tr.ops;
+
+				self.stats.add_writes(ops.len() as u64);
+				self.stats.add_transactions(1);
+
 				for op in ops {
 					// remove any buffered operation for this key
 					self.overlay.write()[op.col() as usize].remove(op.key());
 
 					let cf = cfs.cf(op.col() as usize);
 
+					self.stats.add_write_bytes(match op {
```

Comment:
Would be better to accumulate the values here and call `add_write_bytes` outside of the loop